utoipa-gen/src/openapi/info.rs

```diff
@@ -100,18 +100,18 @@ impl Parse for Info<'_> {
                     info.description =
                         Some(parse_utils::parse_next(input, || input.parse::<Str>())?)
                 }
-                "licence" => {
-                    let licence_stream;
-                    parenthesized!(licence_stream in input);
-                    info.license = Some(licence_stream.parse()?)
+                "license" => {
```

Comment:
I believe this was a bug in https://github.com/juhaku/utoipa/pull/400 , but there hasnt been a release since then.

Reply:
Yes indeed, nice catch